src/gatsby/src/styles/search.scss

```diff
@@ -1160,7 +1068,7 @@ div[class$="-singleValue"] {
   }
 
   .close-icon:hover {
-    color: $colorGrey !important;
+    color: $colorBackgroundGrey !important;
```

Comment:
Did you mean to use a background color variable for a non-background element?

Reply:
Good catch! Actually, this class was not needed so I removed it.